sechub-developertools/scripts/container/postgres/run.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+# SPDX-License-Identifier: MIT
+
+debug () {
+    while true
+    do
+	    echo "Press [CTRL+C] to stop.."
+	    sleep 5s
```

Comment:
Please increase this value to 2-5 minutes.

Reply:
The funny thing is, that CTRL+C only works after the time period. When I want to interrupt my program I have to wait the sleep time...

So I would suggest to keep it as is... if you want to stop it in less then 2 minutes...

Reply:
Set  to 2m, DONE